### DISCUSSION TITLE
Vote for BaymaxHWY as RisingLight member

### DISCUSSION THREAD

**Discussion by skyzh** [risinglightdb/risinglight#627] [Member]
@BaymaxHWY has made great contributions to the RisingLight in the past two months. 

Their contribution begins from the storage part. In #555, @BaymaxHWY added a new functionality to delete files when dropping tables. In #556, they refactored the block cache to support multiple in-flight I/Os. Then, they worked on executors and optimized nested loop join a lot in #567 and #599. Recently, they added composite sort key support in storage merge iterator in #622.

They also found a lot of bugs in RisingLight, which definitely helped RisingLight to become more and more stable.

@BaymaxHWY demonstrated a good understanding of the RisingLight project. Therefore, I'd like to promote them as RisingLight member.

If you are a member of the RisingLight organization, you may vote for this proposal with 👍 and 👎. According to [GOVERNANCE](https://github.com/risinglightdb/risinglight/blob/main/GOVERNANCE.md), a minimum requirement of 2 votes are required for accepting this proposal. Thanks!

**Comment by skyzh** [comment]
Well, GitHub's new poll doesn't support show who voted. I'd lock this discussion and start a new one in the original member section :)

**Comment by skyzh** [comment]
With 3 votes from @skyzh @xxchan @likg227, we are accepting this vote. Welcome to the RisingLight community! The invitation email should have been sent. And @BaymaxHWY will be granted write access to all our repos.

**Reply by BaymaxHWY** [reply]
Thanks! I'll continue to contribute for RisingLight community.
